pixels-common/src/main/resources/pixels.properties

```diff
@@ -155,7 +155,7 @@ executor.selectivity.enabled=true
 # the number of threads used in each worker
 executor.intra.worker.parallelism=8
 # which cloud function service to use, can be lambda (AWS Lambda).
-executor.function.service=lambda
```

Comment:
Don't change the default value 'lambda', but add 'vhive' to the above comment.

Reply:
Okay, I restore the default value and add vhive to comment

Reply:
Mention it can be vhive in the comment.